Naloga3/src/Naloga3.jl

```diff
@@ -0,0 +1,233 @@
+module Naloga3
+
+using LinearAlgebra
+using Plots
+
+
+"""
+    f(t, y, g, l)
+
+Desna stran diferencialnega sistema za matematicno nihalo.
+Pretvorimo drugo enacbo v sistem dveh enacb prvega reda:
+
+    y[1] = theta (odmik),
+    y[2] = omega = dtheta/dt (kotna hitrost)
+
+Sistem:
+    dy1/dt = y2
+    dy2/dt = -(g / l) * sin(y1)
+"""
+function f(t, y, g, l)
+    return [y[2], -(g / l) * sin(y[1])]
+end
+
+
+"""
+    DOPRI(f, y0, tspan, h, g, l)
+
+Resi sistem y' = f(t,y) z metodo Dormand–Prince 5. reda (DOPRI5).
+Implementacija uporablja Butcherjevo tabelo in genericno zanko.
```

Comment:
Lahko bi omenili, da je metoda adaptivna in prilagaja korak, glede na ocenjeno lokalno napako.